src/overlays/actors/ovl_En_Sw/z_en_sw.c

```diff
@@ -679,16 +688,16 @@ void func_80B0DC7C(EnSw* this, PlayState* play) {
     }
 }
 
-s16 func_80B0DE34(EnSw* this, Vec3f* arg1) {
+s16 EnSw_GetRotateY(EnSw* this, Vec3f* target) {
     s16 pitch;
     s16 yaw;
 
-    yaw = Math_Vec3f_Yaw(&this->actor.world.pos, arg1) - this->actor.wallYaw;
-    pitch = Math_Vec3f_Pitch(&this->actor.world.pos, arg1) - 0x4000;
+    yaw = Math_Vec3f_Yaw(&this->actor.world.pos, target) - this->actor.wallYaw;
+    pitch = Math_Vec3f_Pitch(&this->actor.world.pos, target) - 0x4000;
     return pitch * (yaw >= 0 ? -1 : 1);
```

Comment:
I'm not sure exactly what this computes, but it returns more of a pitch (more or less "x rotation") than a yaw (more or less "y rotation")